lib/util/versioning.js

```diff
@@ -55,6 +69,8 @@ function get_runtime_abi(runtime, target_version) {
     }
     if (runtime === 'node-webkit') {
         return get_node_webkit_abi(runtime, target_version || process.versions['node-webkit']);
+    } else if (runtime === 'electron') {
+        return get_electron_abi(runtime, target_version || process.versions.electron);
```

Comment:
ya, id be happy to. i might spend a little time looking into it tonight.  i'd really like to give back somehow to a project that has helped me so much!